src/studio/src/designer/frontend/ux-editor/utils/language.ts

```diff
@@ -97,7 +97,7 @@ export function getCollapsableMenuTitleByType(menu: CollapsableMenus, language:
 }
 
 export function truncate(s: string, size: number) {
-  if (s.length > size) {
+  if (s && s.length > size) {
```

Comment:
It appears that this conditional _was_ correct given the type (`s: string`). It's likely that some value passing through `renderSelectTextFromResources` is not type checked. Given the function's signature, I believe it is likely that this fix will be undone at some point in the future when someone looks at it and decides to clean it for "superfluous" logic.